server/s3_object_metadata.cc

```diff
@@ -35,15 +35,24 @@
 
 extern struct m0_uint128 global_instance_id;
 
-void S3ObjectMetadata::initialize(bool ismultipart, std::string uploadid) {
```

Comment:
you can have this renamed to initialize_destination

Reply:
I think to keep this as is, as this is generic class, and is consumed with GET action classes as well !!! Do you agree ?